chapters/synchronous.tex

```diff
@@ -1103,59 +1103,60 @@ \section{Clocked Discretized Partition}\label{continuous-time-equations-in-clock
 With the language elements defined in this section, continuous-time equations can be used in clocked partitions.
 Hereby, the continuous-time equations are solved with the defined integration method between clock ticks.
 
-From the view of the continuous-time partition, the clock ticks are not interpreted as events, but as step-sizes of the integrator that the integrator must exactly hit.
+Such a partition is called a \willintroduce{discretized partition}, and the clock ticks are not interpreted as events, but as step-sizes of the integrator that the integrator must exactly hit.
```

Comment:
Since we are changing this, can we also change "must exactly hit":
```suggestion
Such a partition is called a \willintroduce{discretized partition}, and the clock ticks are not interpreted as events, but as step-sizes of the integrator that the integrator must hit exactly.
```